app/views/application/ApplicationAccessRequestsView.scala.html

```diff
@@ -22,21 +22,29 @@
 @import models.application.Application
 @import views.html.components.ApplicationDetailsTable
 @import viewmodels.application.ApplicationSideNavPages
+@import views.html.components.IconsLink
+
 
 @this(
     layout: templates.Layout,
     govukTable: GovukTable,
     accessRequestPanel: AccessRequestPanel,
-    applicationDetailsTable: ApplicationDetailsTable
+    applicationDetailsTable: ApplicationDetailsTable,
+    iconsLink: IconsLink
 )
 
 @(application: Application, accessRequests: Seq[AccessRequest], user: UserModel)(implicit request: Request[?], messages: Messages)
 
+@pendingAccessRequestsCount = @{
+    accessRequests.count(_.status == Pending)
```

Comment:
not sure if you would prefer to use exists instead
```suggestion
    accessRequests.exists(_.status == Pending)
```